### ISSUE TITLE
default ipc path incorrect ?

### ISSUE THREAD

**Issue by euri10** [pipermerriam/web3.py#98]
* Version: 2.7.1
* Python: 2.7/3.4/3.5
* OS: /linux/

### What was wrong?
I have a running node launched with geth, no params
```
In[3]: from web3 import Web3, IPCProvider
In[4]: web3 = Web3(IPCProvider())
In[5]: web3.eth.coinbase
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2885, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-5-951bbb3a23b6>", line 1, in <module>
    web3.eth.coinbase
  File "/usr/local/lib/python3.5/dist-packages/web3-2.7.1-py3.5.egg/web3/eth.py", line 70, in coinbase
    return self.request_manager.request_blocking("eth_coinbase", [])
  File "/usr/local/lib/python3.5/dist-packages/web3-2.7.1-py3.5.egg/web3/providers/manager.py", line 20, in request_blocking
    response_raw = self.provider.make_request(method, params)
  File "/usr/local/lib/python3.5/dist-packages/web3-2.7.1-py3.5.egg/web3/providers/ipc.py", line 86, in make_request
    with get_ipc_socket(self.ipc_path) as sock:
  File "/usr/lib/python3.5/contextlib.py", line 59, in __enter__
    return next(self.gen)
  File "/usr/local/lib/python3.5/dist-packages/web3-2.7.1-py3.5.egg/web3/providers/ipc.py", line 27, in get_ipc_socket
    sock.connect(ipc_path)
  File "/usr/local/lib/python3.5/dist-packages/gevent/_socket3.py", line 294, in connect
    raise error(result, strerror(result))
FileNotFoundError: [Errno 2] No such file or directory
In[6]: web3.currentProvider.ipc_path
Out[6]: '/home/user/ethereum/geth.ipc'
```

debugging the stuff makes me think that in this part in https://github.com/pipermerriam/web3.py/blob/master/web3/providers/ipc.py#L49-L55 there should be a .ethereum instead of ethereum, or I'm smoking and don't use it correctly ?
```
    elif sys.platform.startswith('linux'):
        return os.path.expanduser(os.path.join(
            "~",
            "ethereum",
            testnet,
            "geth.ipc",
        ))
```

on my computer the geth.ipc is there : '/home/user/.ethereum/geth.ipc'

#### Cute Animal Picture

![Cute animal picture](http://media3.s-nbcnews.com/i/streams/2013/September/130923/4B9132043-tdy-130923-panda-baby-03.jpg)

**Comment by pipermerriam** [comment]
I believe you are correct.  Blame my fat-fingering that path being wrong.

Would you like to open a PR fixing this?

**Comment by euri10** [comment]
done the PR here https://github.com/pipermerriam/web3.py/pull/99

**Comment by pipermerriam** [comment]
Thank you very much for this.  This is now available in `2.8.1` via pypi or the releases section of this repo.
